airbyte-integrations/connectors/source-jira/source_jira/manifest.yaml

```diff
@@ -1416,3 +1416,8 @@ check:
     - board_issues
     - issues
     - sprint_issues
+
+concurrency_level:
```

Comment:
Can we add a comment saying that this section is useless for now as Jira as no concurrent streams now the reason being that only datetimebasedcursor without partition_routers are supported

Reply:
Fixed